index.html

```diff
@@ -56,25 +62,226 @@ <h1>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+      </section>
+      <section id="about">
+        <div class="main-about">
+          <h1>About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <div class="social-icons">
+            <a href="" id="connect">LET'S CONNECT</a>
+            <ul id="icons-about">
+              <li>
+                <a href="#">
+                  <img src="./images/github.svg" alt="twitter-logo" />
+                </a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/angelist.svg" alt="medium-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/github.svg" alt="github-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/medium.svg" alt="angelist-logo"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <a class="btn-resume" href="#"> Get My Resume </a>
+          </div>
+        </div>
+
+        <ul class="container-end">
+          <li class="cont-lang">
+            <h2>Languages</h2>
+            <img
+              class="bullet-point"
+              src="./images/chevrondown.svg"
+              alt="chevron button"
+            />
+          </li>
+          <ul class="final-stack">
+            <li class="cont-stack">
+              <img src="./images/js.svg" alt="html-logo" />
+              <h3>Javascript</h3>
+            </li>
+            <li class="cont-stack">
+              <img src="./images/html5.svg" alt="css-logo" />
+              <h3 class="props">CSS</h3>
+            </li>
+            <li class="cont-stack">
+              <img src="./images/css.svg" alt="javascript-logo" />
+              <h3 class="props">Javascript</h3>
+            </li>
+          </ul>
+        </ul>
+        <div class="frame-skills">
+          <ul class="underline-frame">
+            <li class="cont-frame">
+              <h1 class="lan-styles">Frameworks</h1>
+              <img src="./images/chevronright.svg" alt="chevron" />
+            </li>
+          </ul>
+
+          <ul class="underline-skills">
+            <li class="cont-skills">
+              <h1 class="lan-styles">Skills</h1>
```

Comment:
- [ ] Just as I suggest above You should use an `h2 or h3` tag instead of an h1. You are meant to have only 1 `h1` on your page and that should be the main title. By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results. :+1: :+1: